### PR TITLE
Tag Cloud: Remove editor style so editor matches frontend

### PR DESCRIPTION
**Pull request by scruffian**
## Description
This moves the editor styles for Tag Cloud to the frontend, and also removes the color style, so that the editor matches the front end.

## How has this been tested?
Using Emptytheme.

## Screenshots 
Before:
| Editor      | Frontend |
| ----------- | ----------- |
| Screenshot 2021-07-08 at 13 55 20|Screenshot 2021-07-08 at 13 55 06|

After:
| Editor      | Frontend |
| ----------- | ----------- |
|Screenshot 2021-07-08 at 14 35 26|Screenshot 2021-07-08 at 14 35 22|

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/block-library/src/tag-cloud/editor.scss [hunk 1]
```diff
@@ -7,7 +7,6 @@
 	span {
 		display: inline-block;
 		margin-left: 5px;
-		color: $gray-700;
 		text-decoration: none;
 	}
 }
```
Comment (youknowriad) [inline packages/block-library/src/tag-cloud/editor.scss]: What about the rest of the styles in this file, are they needed?

Comment (youknowriad) [reply packages/block-library/src/tag-cloud/editor.scss]: Feels some of them could be considered to be moved to `style.css` to have a slightly more opinionated block in terms of spacing.

Comment (kjellr) [reply packages/block-library/src/tag-cloud/editor.scss]: Yeah, I think the rest of the styles here might make sense to apply to the front-end. 

I definitely agree with removing the gray color, but I think the non-underlined number and extra padding we have in the editor is a more polished visual treatment. I'd like to see that applied in both places.

Comment (scruffian) [reply packages/block-library/src/tag-cloud/editor.scss]: Good idea, PR updated...

**Review by kjellr — APPROVED** [review]
This looks great, thanks! Looks the same in both the editor and front-end now:

Screen Shot 2021-07-08 at 11 36 17 AM

File: packages/block-library/src/editor.scss [hunk 2]
```diff
@@ -42,7 +42,6 @@
 @import "./social-links/editor.scss";
 @import "./spacer/editor.scss";
 @import "./table/editor.scss";
-@import "./tag-cloud/editor.scss";
```
Comment (youknowriad) [inline packages/block-library/src/editor.scss:45]: The more we remove these editor.scss files, the better is for parity :)

**Review by youknowriad — APPROVED** [review]

**Pull request merged**